README.md

```diff
@@ -76,6 +76,11 @@ func TestBasicAssertions(t *testing.T) {
 	should.Contain(t, users, "Alice")
 	should.NotContain(t, users, "David")
 	should.Contain(t, userIDs, targetID, should.WithMessage("User ID must exist in the system"))
+
+	// Sort check
+	should.BeSorted(t, []int{1, 2, 3, 4, 5})
```

Comment:
I'm not sure about the need.

I mean this could be used.

should.BeTrue(slices.IsSorted())

https://pkg.go.dev/slices#IsSorted

The error message reporting that x before y doesn't seem useful. It's an assertion library, not a validation one. If tests fails it's enough.

Also using IsSorted implies ascending order, but people will have other needs.

So you would have to add option, or add something close to

https://pkg.go.dev/slices#IsSortedFunc

But also BeNotSorted

I feel like it's more about thinking about a custom function directly